docs/user/troubleshooting/first-troubleshooting-guide.md

```diff
@@ -0,0 +1,21 @@
+# Istio troubleshooting guide
+
+If you are experiencing issues with the cluster, you can follow the steps below to troubleshoot the problem before you contact us. This guide is designed to help you identify the root cause of the problem which might not necessarily be related to the Istio itself but rather to the misconfiguration of the Istio Custom Resources or other network related resources.
+
+## Try to go through the following steps:
+
+1. When an Istio problem is reported for a Kyma module, SRE should first check whether it also occurs for other modules. If the problem only occurs with a specific module, the team that owns that module should start investigating first before involving us.
+2. For connection issues, before forwarding it to Istio module team check [Calico NetworkPolicies](https://docs.tigera.io/calico/latest/network-policy/get-started/calico-policy/calico-network-policy) first. Try to find custom configuration which may affect the connectivity.
```

Comment:
I think we should link to more generic [K8s NetworkPolicies](https://kubernetes.io/docs/concepts/services-networking/network-policies/), since this is what we saw in the past.